spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
```

Comment:
Thanks for your feedback @tomas-stefano. First I will address the better option:

```
Or even better, you can move the timestamp call to the renderer object and then your acceptance tests cover it.
```

The timestamped filename warranted moving this feature to the controllers where it can be dynamically generated before being passed to the CSV renderer. If the filename was included as part of the acceptance test it will be awkward and unwieldy to test for past, present and future scenarios. 

```
scenario 'permits download as CSV' do
    and_there_exists_registered_advisors
    when_i_visit_the_registered_adviser_page
    and_click_on_download_as_csv_button
    then_i_should_have_a_file_with_all_registered_advisers
  end
```

The scenario above will have to be repeated to cater for different time references as previously mentioned.